ui/src/features/get-tests.js

```diff
@@ -271,6 +309,7 @@ class getTests extends React.Component {
             autoHideDuration={4000}
             onRequestClose={this.handleSnackbarClose}
           />}
+          {this.state.testActionError.isError && <ActionErrorPopup onClose={this.resetTestActionError} message={this.state.testActionError.errorMessage} />}
           {error && <ErrorDialog closeDialog={this.onCloseErrorDialog} showMessage={error} />}
```

Comment:
as you can see we already have a component for error.
also you can use the variable error that already exist.
example:
const error = your error ||  errorOnDeleteTest ;

Reply:
@manorlh make sense. done!